src/rpc/net.cpp

```diff
@@ -133,6 +134,10 @@ static UniValue getpeerinfo(const JSONRPCRequest& request)
                                                               "When a message type is not listed in this json object, the bytes received are 0.\n"
                                                               "Only known message types can appear as keys in the object and all bytes received of unknown message types are listed under '"+NET_MESSAGE_COMMAND_OTHER+"'."}
                             }},
+                            {RPCResult::Type::STR, "connection_type", "Type of connection: outbound full relay (default automatic connections),\n"
```

Comment:
```suggestion
                            {RPCResult::Type::STR, "connection_type", "Type of connection:\n " + Join(foobar, ",\n") + "."},
```

What about moving the connection type documentation into a global like `NET_PERMISSIONS_DOC`?

Reply:
done